src/app/modules/main/module.nim

```diff
@@ -1924,7 +1924,7 @@ method activateStatusDeepLink*[T](self: Module[T], statusDeepLink: string) =
   let urlData = self.sharedUrlsModule.parseSharedUrl(statusDeepLink)
   if urlData.notASupportedStatusLink:
     # Just open it in the browser
-    openDefaultBrowser(statusDeepLink)
+    discard openUrl(statusDeepLink)
```

Comment:
Other possibility would be to just request opening given url to the client (UI side). UI could handle that signal e.g. by `Qt.openUrlExternally`. Then there is no need to touch `DOtherSide to do that.

Reply:
Good point! Makes sense to me!

Reply:
@micieslak Updated. Please have a look